src/form-engine.test.tsx

```diff
@@ -598,14 +598,6 @@ describe('Form engine component', () => {
   });
 
   function renderForm(formUUID, formJson, intent?: string) {
-    render(
-      <FormEngine
-        formJson={formJson}
-        formUUID={formUUID}
-        patientUUID={patientUUID}
-        formSessionIntent={intent}
-        visit={visit}
-      />,
-    );
+    render(<FormEngine formJson={formJson} formUUID={formUUID} patientUUID={patientUUID} formSessionIntent={intent} />);
```

Comment:
What happens if its RDE? How do you know the visit  to use if the visit uuid or visit is not passed as a prop?

Reply:
I am not entirely sure what RDE is, please explain. For the second part, [the vistit is being fetched using the patientUuid](https://o3-docs.openmrs.org/docs/framework-api-reference#usevisit:~:text=%E2%96%B8%20useVisit(patientUuid)%3A%20VisitReturnType).